features/works/work_view.feature

```diff
@@ -42,3 +42,20 @@ Feature: View a work with various options
     And I should see "DeletedChapterWork"
     And I follow "Site Map"
   Then I should not see "Sorry, we couldn't find the chapter you were looking for."
+
+  Scenario: other users cannot collect a work by default
+  Given the work "Whatever"
+    And I am logged in as the author of "Whatever"
+    And I have the collection "test collection" with name "test_collection"
+  When I am logged in as "moderator"
+    And I view the work "Whatever"
+  Then I should not see a link "Invite To Collections"
+
+  Scenario: other users can collect a work when the creator has opted-in
+  Given the work "Whatever"
+    And I am logged in as the author of "Whatever"
+    And I set my preferences to allow collection invitations
+    And I have the collection "test collection" with name "test_collection"
+  When I am logged in as "moderator"
```

Comment:
Let's not use "moderator" here, since this is just a random user with no ties to the collection.

Reply:
I might be confusing things, but I think the user _is_ tied to the collection (and needs to be):
* The `And I have a collection` step creates a collection belonging to `moderator`: https://github.com/otwcode/otwarchive/blob/master/features/step_definitions/collection_steps.rb#L65
* The `Invite to Collections` link should only be visible if the current user is a collection maintainer: https://github.com/brianjaustin/otwarchive/blob/AO3-6118-collection-invitation-opt-in/app/views/comments/_commentable.html.erb#L40

If that's right, then the wording/order is a bit weird. Maybe the following would be slightly better?
```cucumber
    And I am logged in as the author of "Whatever"
    And I set my preferences to allow collection invitations
  When I have the collection "test collection" with name "test_collection"
    And I am logged in as "moderator"
```

Reply:
Sounds good!